api/src/main/scala/org/virtuslab/ideprobe/protocol/AwaitIdle.scala

```diff
@@ -0,0 +1,9 @@
+package org.virtuslab.ideprobe.protocol
+
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+
+case class AwaitIdle(initialWait: FiniteDuration, newTaskWait: FiniteDuration, checkFrequency: FiniteDuration)
```

Comment:
Maybe name it AwaitIdleParams, I think there is similar situation with building (check the class name there)

Reply:
There are BuildParams